### PR TITLE
Further simplification of GamesPanel + Emulator tests

### PR DESCRIPTION
**Pull request by JeremyHurwitz**

### CODE REVIEW

File: src/app/FirebaseUtil.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import firebase from 'firebase';
+
+let firebaseInstance: firebase.app.App | null = null;
```
Comment (jyoo84) [inline src/app/FirebaseUtil.ts:3]: This merits some comments.

Comment (jcavanagh) [reply src/app/FirebaseUtil.ts:3]: This defines the type of `firebaseInstance` as either a `firebase.app.App` or `null`.  

Typescript does not allow null to be assigned unless the type is explicitly nullable, or strict null checking is disabled.

Comment (JeremyHurwitz) [reply src/app/FirebaseUtil.ts:3]: I think Juhwan was referring to the dependency injection, not the TS syntax.

**Comment by JeremyHurwitz** [comment]
I got the tests passing. Sadly, there are still a few warnings when running the tests (different warnings locally vs GitHub Actions).

**Comment by JeremyHurwitz** [comment]
I'm calling this good enough. The warnings are beyond me.

**Pull request merged**